src/main/java/tc/oc/pgm/rotation/PGMMapOrder.java

```diff
@@ -1,15 +1,40 @@
 package tc.oc.pgm.rotation;
 
+import tc.oc.pgm.api.match.Match;
 import tc.oc.pgm.map.PGMMap;
 
 /**
- * An order of {@link PGMMap}s, which can either be a {@link FixedPGMMapOrder} or a {@link
- * RandomPGMMapOrder}
+ * A provider of {@link PGMMap} ordering order. It is responsible for providing the next map the
+ * server should play.
  */
 public interface PGMMapOrder {
+  /**
+   * Returns next map to play, and removes it from the queue.
+   *
+   * @return The next map to play
+   */
   PGMMap popNextMap();
 
+  /**
+   * Returns the next map that will be played. Returning a null value is allowed and specifies the
+   * map order hasn't picked the next map yet.
+   *
+   * @return The next map to play, if present.
+   */
   PGMMap getNextMap();
 
+  /**
+   * Forces a specific map to be played next. The underlying PGMMapOrder may ignore this, but it is
```

Comment:
Use `{@link PGMMapOrder}` helps with renaming in the future.

Reply:
Fixed both there and on another comment